packages/nextjs/app/builders/0x6fdee3eFB06d0AdffaA5363307D3cA3C1c73F476/page.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import Link from "next/link";
+import type { NextPage } from "next";
+import { FaGithub } from "react-icons/fa";
+import { FaLinkedin } from "react-icons/fa";
+import { FaXTwitter } from "react-icons/fa6";
+
+const PortfolioBijoy: NextPage = () => {
+  return (
+    <>
+      <div className="flex flex-col items-center">
+        <div className="flex flex-col mt-24 items-center">
+          <img src="/luffy.jpg" alt="" className="rounded-lg w-36 h-36" />
```

Comment:
Your image is a horizontal rectangle, but setting both width and height is squeezing it into a square.
Also, you probably want to use [Next's Image Component](https://nextjs.org/docs/pages/api-reference/components/image).